src/normalize.js

```diff
@@ -15,7 +15,7 @@ const extractFields = async (
       // add recursion to fetch nested strapi references
       await Promise.all(
         field.map(async f =>
-          extractFields(apiURL, store, cache, createNode, touchNode, auth, f)
+          await extractFields(apiURL, store, cache, createNode, touchNode, auth, f)
```

Comment:
Here this is not necessary because it is wrapped in a Promise.all. You can even remove the async keyword.